src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -103,10 +103,25 @@ internal class RpcFunctionInvocationDispatcher : IFunctionInvocationDispatcher
 
         internal ConcurrentStack<WorkerErrorEvent> LanguageWorkerErrors => _languageWorkerErrors;
 
-        internal int MaxProcessCount => _maxProcessCount;
-
         internal IWebHostRpcWorkerChannelManager WebHostLanguageWorkerChannelManager => _webHostLanguageWorkerChannelManager;
 
+        private async Task<int> GetMaxProcessCount()
+        {
+            if (_workerConfigs != null
+                && _environment != null
+                && _workerConcurrencyOptions != null
```

Comment:
move null ref checks on _evironment and _workerconfigs to constructor - if these are null - DI setup is has gone wrong!

Reply:
great point, ill move!